docs/user-guide/vfile-plugin.md

```diff
@@ -3,14 +3,14 @@ title: vFile volume plugin for Docker
 
 ---
 ## Overview
-Depending on the underlying block storage device system, it might not be possible to access the same 
+Depending on the underlying block storage device system, it might not be possible to access the same
 persistent volume across different hosts/nodes simultanously.
 For example, currently users cannot mount the same persistent volume which is created through
 vSphere Docker Volume Service (vDVS) on containers running on two different hosts at the same time.
 
 This can be solved through distributed file systems, such as NFS, Ceph, Gluster, etc.
-However, setting up and maintaining those distributed file systems for docker persistent data usage is not a trivial work. 
-Furthermore, users can face more challenges in order to achieve high availability, scalability, and load balancing. 
+However, setting up and maintaining those distributed file systems for docker persistent data usage is not a trivial work.
```

Comment:
those distributed file systems for docker persistent data usage -> enterprise storage offerings for Cloud Native usecases